tools/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/openfga/openfga/tooling
 go 1.18
 
 require (
-	github.com/golang/mock v1.6.0
+	github.com/golang/mock v1.7.0-rc.1
```

Comment:
This adds support to `mockgen` for generics, which was required for this PR because we now generic storage interface mocks that depend on generics.